talend-component-maven-plugin/src/main/java/org/talend/sdk/component/maven/ComponentDependenciesBase.java

```diff
@@ -143,7 +143,7 @@ protected Artifact resolveArtifact(final Artifact artifact, final String classif
     }
 
     protected String computeCoordinates(final Artifact artifact) {
-        return artifact.getGroupId() + ":" + artifact.getArtifactId() + ":" + artifact.getVersion();
+        return artifact.getGroupId() + ":" + artifact.getArtifactId() + ":" + getVersion(artifact);
```

Comment:
Doesn't that remove the `-SNAPSHOT` for snapshots? :thinking: 
Is that OK? 
I'm not sure about the impact of this one :sweat_smile:

Reply:
No, that's the opposite it's doesn't take timestamp but x.y.z-SNAPSHOT. Have a look at issue description.